docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Known compatibility issues with Android apps
+sidebar_position: 14
+---
+
+## Conflict apps
+
+### Various VPN apps
+
+All VPN apps are incompatible with AdGuard running in VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we can suggest you:
```

Comment:
```suggestion
Other VPN apps are incompatible with AdGuard running in the Local VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we suggest you:
```